sources/packages/web/src/components/layouts/Institution/sidebar/ManageInstitutionSideBar.vue

```diff
@@ -11,6 +11,15 @@
         :title="item.label"
       />
     </v-list>
+    <template #append>
+      <v-list density="compact" nav>
```

Comment:
FYI- this might conflict with my PR

Reply:
No problem, I can take yours and just add the menu item change on top of if 😉 
Thanks for the heads up.